irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -151,45 +198,31 @@ namespace {
             % permittee)
         .str();
   }
+
+  template <typename Format>
+  void appendCommandName(const std::string &name,
+                         Format &cmd,
+                         bool do_validation) {
+    auto command_name =
+        name + (do_validation ? "WithValidation" : "WithoutValidation");
```

Comment:
It looks like "WithValidation" and "WithoutValidation" should be defined as constants in struct PreparedStatement, so that they can be accessed here.